### PR TITLE
Get ISO check gpg signatures from an env var

### LINKED ISSUE #199: Implement gpg check toggle feature  in jenkins for iso installation
Add the option to enable or disable GPG checking for ISO installation. Also add the option to related jobs on robottelo-ci.

### PR DESCRIPTION
**Pull request by elyezer**
Get the information from an environment variable in addition to the task
argument to allow jenkins setting it from a job parameter.

Closes #199

### CODE REVIEW

**Comment by elyezer** [comment]
Will link the robottelo-ci PR (will send it later) to set the added variable in this PR.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -790,19 +791,26 @@ def iso_install(
...
-    if isinstance(check_sigs, str):
-        check_sigs = (check_sigs.lower() == 'true')
-
-    if admin_password is None:
-        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
-
     iso_url = os.environ.get('ISO_URL') or os.environ.get('BASE_URL')
     if iso_url is None:
         print('Please provide a valid URL for the ISO image.')
         sys.exit(1)
```
Comment (elyezer) [inline automation_tools/__init__.py]: Let's fail earlier.

**Comment by elyezer** [comment]
robottelo-ci PR sent https://github.com/SatelliteQE/robottelo-ci/pull/38

**Comment by sthirugn** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**